testsupport/tiers/checks.go

```diff
@@ -1227,6 +1239,66 @@ func appstudioViewRoleBinding(userName string) spaceRoleObjectsCheck {
 	}
 }
 
+func appstudioQuotaComputeDeploy() namespaceObjectsCheck { // nolint:unparam
+	return func(t *testing.T, ns *corev1.Namespace, memberAwait *wait.MemberAwaitility, _ string) {
+		var err error
+		spec := corev1.ResourceQuotaSpec{
+			Scopes: []corev1.ResourceQuotaScope{corev1.ResourceQuotaScopeNotTerminating},
+			Hard:   make(map[corev1.ResourceName]resource.Quantity),
+		}
+		spec.Hard[corev1.ResourceLimitsCPU], err = resource.ParseQuantity("2000m")
```

Comment:
```suggestion
		spec.Hard[corev1.ResourceLimitsCPU], err = resource.ParseQuantity("20000m")
```
I guess this should be 20000m instead?

Reply:
Or maybe the host operator PR should have `2000m` instead?